app/api/users.js

```diff
@@ -303,4 +303,94 @@ router.delete('/:userId', requireAuthentication, async function (req, res, next)
   }
 })
 
```

Comment:
maybe have a quick comment atop these 2 handlers explaining what they do, like in the PR?